src/components/MeshMember.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from "react";
+import { MetadataEntry } from "../static/metadata";
+import {
+  shortenPgpOrReturnEmail,
+  formatTwitterHandle
+} from "../utils/attributeUtils";
+
+type Props = {
+  data: MetadataEntry;
+};
+
+const MeshMember: React.FC<Props> = (props) => {
+  const [hovering, setHovering] = useState(false);
+
+  const { data } = props;
```

Comment:
why destructure here and not in the function parameter?

Reply:
alex got me for it in other PR's, normally would